module/VuFind/sql/mysql.sql

```diff
@@ -177,10 +177,12 @@ CREATE TABLE `external_session` (
 CREATE TABLE `shortlinks` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
   `path` mediumtext NOT NULL,
+  `hash` varchar(9) NOT NULL,
```

Comment:
Do we need to add some kind of default value here so that when the column is initially added to a table containing existing data, the database doesn't reject the change based on constraints? (I haven't tested what happens if you try this, but I suspect that between the "NOT NULL" and the lack of default and the UNIQUE index, there may be problems for the automatic database upgrade process.

Reply:
:+1: